nsx_vtep/actions/nsxraml/html-version/readme.md

```diff
@@ -0,0 +1,19 @@
+# HTML Version of the NSX RAML Specification
```

Comment:
Does the pack actually need all of this nsxraml content? Or just some specific files/folders?